utilities/src/main/java/io/syndesis/qe/bdd/validation/DbValidationSteps.java

```diff
@@ -310,4 +311,18 @@ private String getLeadTaskFromDb() {
         }
         return leadTask;
     }
+
+    @Then("^check rows number of table \"([^\"]*)\" is greater than (\\d+) after (\\d+) s$")
+    public void checkRowsNumberIsGreaterThan(String table, int treshold, int s) throws InterruptedException, SQLException {
+        Thread.sleep((long)(s*1000 + 1000));
```

Comment:
this can be rewritten as `Thread.sleep(s*1000L)`

Reply:
done